SwiftKeychainWrapperTests/KeychainWrapperTests.swift

```diff
@@ -126,7 +126,7 @@ class KeychainWrapperTests: XCTestCase {
     }
     
     func testStringRetrievalWhenValueDoesNotExist() {
```

Comment:
I think the body should be replaced with just this

```
let retrievedString = KeychainWrapper.stringForKey(testKey)
XCTAssertNil(retrievedString, "String for Key should not exist")
```

There's no point in asserting on `true` and if you discard the value with `_` you won't be able to observe it in the debugger in case the test fails some day.